### PR TITLE
Update peer storage path config

### PR DESCRIPTION
**Pull request by thibault-martinez**
## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that new and existing unit tests pass locally with my changes
- [x] I have updated the CHANGELOG.md, if my changes are significant enough

### CODE REVIEW

File: bee-api/bee-rest-api/Cargo.toml [hunk 1]
```diff
@@ -11,15 +11,15 @@ keywords = [ "iota", "tangle", "bee", "framework", "api" ]
...
+bee-message = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false }
+bee-pow = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false }
 bee-protocol = { path = "../../bee-protocol", default-features = false }
-bee-runtime = { git = "https://github.com/iotaledger/bee.git", branch = "dev", default-features = false, optional = true  }
-bee-storage = { git = "https://github.com/iotaledger/bee.git", branch = "dev", default-features = false, optional = true }
-bee-tangle = { git = "https://github.com/iotaledger/bee.git", branch = "dev", default-features = false, optional = true }
+bee-runtime = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false, optional = true  }
+bee-storage = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false, optional = true }
+bee-tangle = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false, optional = true }
```
Comment (Alex6323) [inline bee-api/bee-rest-api/Cargo.toml]: reminder

File: bee-node/Cargo.toml [hunk 2]
```diff
@@ -11,19 +11,19 @@ keywords = [ "iota", "tangle", "bee", "framework", "node" ]
...
-bee-storage-sled = { git = "https://github.com/iotaledger/bee.git", branch = "dev", default-features = false, optional = true }
-bee-tangle = { git = "https://github.com/iotaledger/bee.git", branch = "dev", default-features = false }
+bee-runtime = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false }
+bee-storage = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false }
+bee-storage-null = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false }
+bee-storage-rocksdb = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false, optional = true }
+bee-storage-sled = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false, optional = true }
+bee-tangle = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false }
```
Comment (Alex6323) [inline bee-node/Cargo.toml]: reminder

File: bee-protocol/Cargo.toml [hunk 3]
```diff
@@ -11,15 +11,15 @@ keywords = [ "iota", "tangle", "bee", "framework", "protocol" ]
...
+bee-common = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false, optional = true }
+bee-gossip = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false }
+bee-ledger = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false, features = [ "workers" ], optional = true }
+bee-message = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false, features = [ "serde" ] }
+bee-pow = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false }
+bee-runtime = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false, optional = true }
+bee-storage = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false, optional = true }
+bee-tangle = { git = "https://github.com/thibault-martinez/bee.git", branch = "add-peer-storage-path", default-features = false, optional = true }
```
Comment (Alex6323) [inline bee-protocol/Cargo.toml]: reminder

File: bee-node/src/entrynode/builder.rs [hunk 4]
```diff
@@ -223,7 +223,10 @@ async fn initialize_autopeering(
     let mut peerstore_options = RocksDbPeerStoreConfigOptions::default();
     peerstore_options.create_if_missing(true);
     peerstore_options.create_missing_column_families(true);
-    let peerstore_cfg = RocksDbPeerStoreConfig::new(PEERSTORE_PATH, peerstore_options);
+    let peerstore_cfg = RocksDbPeerStoreConfig::new(
+        builder.config().autopeering_config.peer_storage_path(),
+        peerstore_options,
```
Comment (Adam-Gleave) [inline bee-node/src/entrynode/builder.rs:228]: Should this be renamed as well now?

Comment (Alex6323) [reply bee-node/src/entrynode/builder.rs:228]: there we go :wink:

Comment (thibault-martinez) [reply bee-node/src/entrynode/builder.rs:228]: I will keep it for now if you allow me, I have some node refactoring/cleaning coming up soon

Comment (thibault-martinez) [reply bee-node/src/entrynode/builder.rs:228]: But yes, it should

Comment (Alex6323) [reply bee-node/src/entrynode/builder.rs:228]: I'ld be happy I don't need to rename everything to `peerstorage` now.

**Pull request merged**